tests/platforms/android/gradle/test_run.py

```diff
@@ -211,12 +267,18 @@ def mock_stream_output(app, stop_func, **kwargs):
         f"{first_app_config.package_name}.{first_app_config.module_name}",
     )
 
+    assert run_command.tools.mock_adb.forward.call_count == 0
+    assert run_command.tools.mock_adb.reverse.call_count == 0
```

Comment:
`assert_not_called` is better, because when the assertion fails, it prints details of the calls that were made.

```suggestion
    run_command.tools.mock_adb.forward.assert_not_called()
    run_command.tools.mock_adb.reverse.assert_not_called()
```